Source/Core/Common/CDUtils.cpp

```diff
@@ -208,7 +208,7 @@ std::vector<std::string> cdio_get_devices()
 // Returns true if device is a cdrom/dvd drive
 bool cdio_is_cdrom(std::string device)
 {
-#ifndef _WIN32
+#ifdef _WIN32
```

Comment:
When I tried to compile it, it treated that ifndef as if were not there.  I thought the simplest correct fix was to change that ifndef to an ifdef.